README.md

```diff
@@ -4,9 +4,9 @@ This is a living document that can serve as a resource for finding people includ
 
 This resource began in reponse to a discussion post started by [Christine Belzie](https://github.com/CBID2) titled [Where are all the women owned Open Source projects](https://dev.to/cbid2/where-are-all-the-women-owned-open-source-projects-4pd0). In looking for some resources to send Christine, I was unable to find anything thorough or currently maintained and decided to start this project.  It is our hope that this resource will get so large that we will have to find a better solution to hold all the amazing resources.
 
-Contributions to this resource are welcome and encouraged. 
+Contributions to this resource are welcome and encouraged.
 
-When adding someone please add their name, link to their github profile, project name, and link to project.
+Click on the [CONTRIBUTING.md](https://github.com/amandamartin-dev/women-led-open-source/blob/main/CONTRIBUTING.md) to learn how to submit resources. 
```

Comment:
Is this one ok @amandamartin-dev? I thought linking the contributing file would make it easier for readers to learn how to contribute.

Reply:
oh yes! 

also, just had another thought about the other section - I think that could be useful in the contributing.md as we didn't add the context there to remind folks what types of ppl we want to highlight in this resource. wdyt?

Reply:
I like that! :)

Reply:
> oh yes!
> 
> also, just had another thought about the other section - I think that could be useful in the contributing.md as we didn't add the context there to remind folks what types of ppl we want to highlight in this resource. wdyt?

Made an issue for it: https://github.com/amandamartin-dev/women-led-open-source/issues/16